openc3-cosmos-init/plugins/packages/openc3-vue-common/src/tools/base/Login.vue

```diff
@@ -64,7 +64,7 @@
           <v-row>
             <v-btn
               type="submit"
-              @click.prevent="verifyPassword"
+              @click.prevent="() => verifyPassword()"
```

Comment:
How does this work since verifyPassword takes 2 parameters?

Reply:
All js function arguments default to `undefined` if you don't pass them in. So for when it's called from here, the `||=` operator on line 163 will set `token`, and the `!` operator on line 181 will negate `undefined` to set `this.showAlert` to true.

This line could be written as `@click.prevent="() => verifyPassword(password, false)"` (or even `verifyPassword(null, false)`) but it's unnecessary to be explicit